example/ck_tile/03_gemm/instances/gemm_api.cpp

```diff
@@ -57,204 +18,188 @@ float gemm(const gemm_traits& t, const ck_tile::GemmHostArgs& a, const ck_tile::
         {
             if(a.M > 512)
             {
-                // universal gemm compute bound RR
-                std::cout << "fp16 comp\n";
-                return gemm_<trait_<FP16,
-                                    FP16,
-                                    FP32,
-                                    FP16,
-                                    Row,
-                                    Row,
-                                    Row,
-                                    256,
-                                    256,
-                                    32,
-                                    2,
-                                    2,
-                                    1,
-                                    32,
-                                    32,
-                                    16,
-                                    false,
-                                    false,
-                                    false>>(a, s);
+                return gemm_<gemm_traits_<FP16,
+                                          FP16,
+                                          FP32,
+                                          FP16,
+                                          Row,
+                                          Row,
+                                          Row,
+                                          256,
+                                          256,
+                                          32,
+                                          2,
+                                          2,
+                                          1,
+                                          32,
+                                          32,
+                                          16,
+                                          false,
+                                          false,
+                                          false>>(a, s);
```

Comment:
Please turn off clang format also here. It would be beneficial to have `//` comment, as we used to have in old ck instances, describing what specific value referes to.